src/views/SystemProfile/components/Team/Edit/TeamMemberForm.tsx

```diff
@@ -212,8 +284,9 @@ const TeamMemberForm = ({
               )}
               <MultiSelect
```

Comment:
Just noticed that in the Figma, the label for the selected roles is "Selected role(s)", but it currently says "Selected options". It's not necessarily part of this ticket, but would you mind fixing this?

The `selectedLabel` prop in the `MultiSelect` component sets this text.

Reply:
@adamodd I think [this](https://github.com/CMS-Enterprise/easi-app/blob/a485947c85583c7dfb3daa925e2b58c353980fcf/src/components/shared/MultiSelect/index.tsx#L254-L256) is where the [prop](https://github.com/CMS-Enterprise/easi-app/blob/a485947c85583c7dfb3daa925e2b58c353980fcf/src/components/shared/MultiSelect/index.tsx#L144) is used in the MultiSelect

I think we should just be able to pass a translated key from i18n through that prop to achieve this (and I think we might also already have that key [here](https://github.com/CMS-Enterprise/easi-app/blob/a485947c85583c7dfb3daa925e2b58c353980fcf/src/i18n/en-US/systemProfile.ts#L245))

Reply:
I think 845fcf387 should resolve this!